plugins/Annotations/Controller.php

```diff
@@ -218,4 +225,31 @@ public function getEvolutionIcons()
 
         return $view->render();
     }
+
+    /**
+     * Private helper method to get user aliases from user logins
+     *
+     * Query Param Input:
+     *  - annotations: List of annotations to get user aliases for  
+     *
+     * Output:
+     *  - List of users with logins and aliases
+     */
+    private function getAnnotationUsers($annotations = '')
```

Comment:
You should use an empty array as default for `$annotations`, otherwise the foreach below might fail